bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/PageImpl.java

```diff
@@ -89,6 +91,7 @@ protected void initModel() {
             }
         }
         setRedirect();
+        populateHeadClientlibCategories();
```

Comment:
this one is picky, but could we move this before `setRedirect` so we have the clientlib related logic together.